build_stats.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+
+set -e
+
+HERE_DIR="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
```

Comment:
This breaks for me in two ways. Firstly if I run `sh build_stats.sh` then `HERE_DIR` turns out to be `/bin/` (i.e. where the `sh` binary is stored). 

If I run it as `./build_stats.sh` the script fails (as described above) with:

```
configure: error: expected an absolute directory name for --prefix: ./work/R-inst
```

as it needs a full path for `HERE_DIR` and not just `.`.